web/src/settings_org.ts

```diff
@@ -1320,6 +1321,7 @@ export function build_page(): void {
     meta.loaded = true;
 
     loading.make_indicator($("#admin_page_auth_methods_loading_indicator"));
+    settings_banner.set_up();
```

Comment:
Calling `set_up` here, and from groups and stream settings related file as well, means that "click" handlers are being set up multiple times and that can result in multiple tabs being opened for the same link.